File: IOS/Features/Profile/Badge/Badge.swift
//
//  Badge.swift
//  WatchVerse
//
//  Created by Maxime on 2026-08-10.
//

import Foundation

struct Badge: Identifiable, Equatable, Codable {
    let id: String
    let title: String

    let universeID: String
    let universeTitle: String

    let artwork: String
    let description: String

    let requiredContentIDs: [String]

    let sortOrder: Int
}

File: Android/WatchVerse/app/src/main/java/com/maximeproulx/watchverse/Badge.kt
package com.maximeproulx.watchverse

data class Badge(
    val id: String,
    val title: String,
    val universeID: String,
    val universeTitle: String,
    val artwork: String,
    val description: String,
    val requiredContentIDs: List<String>,
    val sortOrder: Int,
    val isUnlocked: Boolean = false
)
